File: 1535-find-the-winner-of-an-array-game/1535-find-the-winner-of-an-array-game.java
class Solution {
    public int getWinner(int[] arr, int k) {
        int winner = arr[0];
        int c = 0;
        for (int i = 1; i < arr.length; i++) {
            if (arr[i] > winner) {
                c = 1;
                winner = arr[i];
            } else {
                c++;
            }
            if (c == k) {
                return winner;
            }
        }
        return winner;
    
    }
}

File: 1535-find-the-winner-of-an-array-game/1535-find-the-winner-of-an-array-game.py
class Solution(object):
    def getWinner(self, arr, k):
        """
        :type arr: List[int]
        :type k: int
        :rtype: int
        """
        winner=arr[0]
        c=0
        for i in range(1,len(arr)):
            if arr[i]>winner:
                c=1
                winner=arr[i]
            else:
                c+=1
            if c==k:
                return winner
        return winner